src/modules/esl-tooltip/core/esl-tooltip.shape.ts

```diff
@@ -0,0 +1,20 @@
+import type {ESLPopupTagShape} from '../../esl-popup/core/esl-popup.shape';
+import type {ESLTooltip} from './esl-tooltip';
+
+/**
+ * Tag declaration interface of {@link ESLTooltip} element
+ * Used for TSX declaration
+ */
+export interface ESLTooltipTagShape<T extends ESLTooltip = ESLTooltip> extends ESLPopupTagShape<T> {
+  /** Disable arrow at Tooltip */
```

Comment:
```suggestion
  /** Disables arrow at Tooltip */
```

Reply:
It seems to me that we use third-person singular verbs only for methods description, doesn't it?